src/main.rs

```diff
@@ -18,26 +18,167 @@
 //!         ),
 //!     ]
 //! ```
+
 #![deny(missing_docs)]
 #![deny(clippy::all)]
+//#![feature(proc_macro_hygiene, decl_macro)]
 
 mod bundle;
 mod config;
 mod virtfs;
 mod workload;
 
+#[macro_use]
+extern crate serde_derive;
+
+use openssl::asn1::Asn1Time;
+use openssl::hash::MessageDigest;
+use openssl::pkey::PKey;
+use openssl::rsa::Rsa;
+use std::net::{IpAddr, SocketAddr};
+use std::path::Path;
+use warp::Filter;
+#[derive(Serialize, Deserialize)]
+struct Payload {
+    encoding: String,
+    contents: Vec<u8>,
+}
+
 use log::info;
+/// Source of the key to use for TLS
+//pub const KEY_SOURCE: &str = "file-system";
+pub const KEY_SOURCE: &str = "generate";
```

Comment:
Remove this. `wasmldr` should always use a self-signed cert generated at runtime. We'll handle the (slight) exception to this later for SEV.